main/src/ca/uwaterloo/flix/language/errors/LexerError.scala

```diff
@@ -26,20 +26,20 @@ sealed trait LexerError extends CompilationMessage {
 object LexerError {
 
```

Comment:
sort the case classes inside this object (to keep them in the same order as TokenErrorKind).